assigner/backends/decorators.py

```diff
@@ -1,15 +1,25 @@
+import logging
+
 from assigner.backends import GitlabBackend, MockBackend
-from assigner.config import config_context
+from assigner.config import requires_config
+
+
+logger = logging.getLogger(__name__)
 
 
-def require_backend(func):
+def requires_backend_and_config(func):
```

Comment:
NP 9000: Maybe name this `requires_config_and_backend` since it passes `config` first and `backend` second to the functions it wraps? Or we could change the order in which the args are passed.

Reply:
I ran sed for you.